uaa/src/test/java/org/cloudfoundry/identity/uaa/integration/pageObjects/SamlErrorPage.java

```diff
@@ -0,0 +1,16 @@
+package org.cloudfoundry.identity.uaa.integration.pageObjects;
+
+import org.openqa.selenium.WebDriver;
+
+import static org.hamcrest.Matchers.containsString;
+import static org.hamcrest.Matchers.endsWith;
+
+public class SamlErrorPage extends Page {
+    static final protected String urlPath = "/saml_error";
```

Comment:
Is there a reason you make these constants `protected` instead of `private`, even though it appears nothing inherits the class?

I would advocate for the principle of least privilege.

Reply:
I think I had inheritance in mind, but as the limited use of inheritance so far doesn't need it, I have changed all of these constants to private.